docs/user_guide.md

```diff
@@ -131,3 +131,12 @@ Below is the list of features in this panel:
 - The user can click the title again to display the file content.
 - The lines that are NOT written by the current author will be marked gray, but they will still be displayed to provide context for the user
 - Segments of codes that are not written by the current author is default to be collapsed. If the user wants to refer to it, he can open the segment.
+
+## User Dashboard V2 (beta)
+Works is in progress for the second version of the dashboard. Instructions to get it up and running:
+- start a webserver serving your generated report directory
+- `python2 -m SimpleHTTPServer 8000` or `python3 -m http.server 8000`
+- go to `http://localhost:8000/v2` on your browser
+- in the data field, enter `http://localhost:8000` 
```

Comment:
As mentioned earlier, avoid more complexity and dependency on additional software if necessary. 
Also, try to make it deploy-able on static web page - e.g. GitHub pages.
It is okay if Google chrome is the only one that's unable to display the page for now.